panel/chat/message.py

```diff
@@ -498,6 +498,7 @@ def _create_panel(self, value, old=None):
         """
         if isinstance(value, Viewable):
             self._internal = False
+            self._include_stylesheets_inplace(value)
```

Comment:
I'm always a little wary for applying stylesheets inplace on objects the user created but I guess it's okay here.